Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcneilco/centos-r-repo:1.13.6
+FROM mcneilco/acas-r-repo:1.13.7
```

Comment:
We are still using centos but I changed the name of the repo from `centos` -> `acas` because its unnecessary to have in the name.

acas-r-repo:1.13.7 is basically a static image, but I added notes on how it was built to the readme in this repo.